content/blog/2021-09-30-september-21-community-gems.md

```diff
@@ -0,0 +1,127 @@
+---
+title: September '21 Community Gems
+date: 2021-09-30
+description: |
+  A roundup of technical Q&A's from the DVC and CML communities.
+  This month: data registries, working with DVC remotes,
+  queued experiments, and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC and CML community.
+  This month: data registries, working with DVC remotes,
+  queued experiments, and more.
+picture: 2021-09-30/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/september-21-community-gems/871
+tags:
+  - Data Registry
+  - DVC Remotes
+  - Queued Experiments
+  - Pipelines
+  - Git
+---
+
+### [If I already have several projects and want to build a data registry out of those?](https://discord.com/channels/485586884165107732/563406153334128681/879651401318400021)
+
+Great question from @m41n314c!
+
+You can import into the registry from your other projects with `dvc import` and
```

Comment:
Hmm, looks a bit dangerous to me. Do we actually want to recommend this as an answer to this general question? @flippedcoder @dberenbaum ? The downside of this approach is speed (need to resolve two times, clone repos, etc) and security - data is not controlled by the registry (as it is mentioned). Should we include a better recommendation here as an alternative - e.g. move data to a new repo, or import directory from source repos (kinda distributed data registry).

Reply:
Maybe we can just drop this one rather than block this post? I don't think we have established best practices yet for data registry security.

> import directory from source repos (kinda distributed data registry).

What do you mean here? How is it different than the current answer?

> Should we include a better recommendation here as an alternative

I might be missing something, because it's not obvious that the centralized registry is generally better. As mentioned in the discord discussion, distributed access can be useful so that the data owners can manage their data rather than have a central team manage all datasets. It's not clear to me how to easily manage access in the case of a centralized registry.

Reply:
@shcheklein What do you think about including both like I have in the update? I definitely need more help explaining the recommended approach if this line doesn't handle it.

Reply:
(for the record: I'm fine with `Maybe we can just drop this one rather than block this post?` - no need to block anything)

Okay, sorry, I think I got confused by the answer, e.g.

`The recommended and most secure _approach_ ... move your data _or_ import the data from source repos` 

(or was it updated?)

and then we go:

`You can import _into_ the registry` (does it mean that we an intermediate central repo that imports from other repos and then clients import from it? or something else).

So, I'm not sure to be honest at this moment what approach do we recommend in this answer. Could you please clarify - that would help to review and improve this.

Reply:
(for the record: I'm fine with Maybe we can just drop this one rather than block this post? - no need to block anything)

👍 

> (or was it updated?)

Yes, I think this was added based on your earlier comment.

> `You can import _into_ the registry` (does it mean that we an intermediate central repo that imports from other repos and then clients import from it? or something else).

This is what I understood the original answer to mean.

>     import directory from source repos (kinda distributed data registry).

Reading this again, does this mean that there is no "intermediate central repo"?

Reply:
But there was also `You can import _into_ the registry` ?

Reply:
I'm a little confused. 😅 Should I drop this question in favor of a different one or just go with the original answer?

Reply:
@flippedcoder I think you can drop it since there doesn't seem to be a great conclusion about what to put. Not sure if you need to replace it with anything.

@shcheklein `You can import _into_ the registry` was the original answer (I think taken from discord). `The recommended and most secure _approach_ ... move your data _or_ import the data from source repos` was added later (I think taken from your comment).

Reply:
Yep, okay, the way I read it (and I can be completely wrong here) is that `You can import _into_ the registry` is not the same as `import directory from source repos (kinda distributed data registry).`. The first way is a bit more complicated - it's like a chain of imports (that's why I was asking about that intermediate registry and all that stuff).

Reply:
Yup, I got the difference once I saw "intermediate." Still curious what we should recommend here, but we could make that a separate discussion.

Reply:
(Btw if remove/replace this we might need to update the `description` and `descriptionLong`)

Reply:
I think it'll be ok to leave the `description` and `longDescription` the same since there's another data registry question.